src/cli/doc/Cli.md

```diff
@@ -0,0 +1,40 @@
+# CLI
+
```

Comment:
I think you need a few sentences to introduce the topic at a high level.

Reply:
I added 3 sentences.